src/exports.js

```diff
@@ -17,13 +17,26 @@ type Extension = {
 export type MaybeSheetDefinition = SheetDefinition | false | null | void
 */
 
+// True to minify classnames.
+// False to not minify classnames.
+// Unset to minify only in 'production' environment
+let forceMinify = undefined;
+const shouldMinify = () => {
+    if (forceMinify !== undefined) {
+        return forceMinify;
+    } else {
+        return process.env.NODE_ENV === 'production';
+    }
+}
+
+
 const StyleSheet = {
     create(sheetDefinition /* : SheetDefinition */) {
         return mapObj(sheetDefinition, ([key, val]) => {
             const stringVal = JSON.stringify(val);
             return [key, {
                 _len: stringVal.length,
-                _name: process.env.NODE_ENV === 'production' ?
+                _name: shouldMinify() ?
```

Comment:
With the previous version of this code, the conditional will end up being minified out in production, which gives us a small performance boost. Switching this to a runtime check will affect performance some amount.

It is probably a pretty small hit, but given that this code is in a pretty performance-sensitive path, it is definitely worth profiling the fully simplified version (e.g. `_name: hashString(stringVal)`) against this version in real-world scenarios so we at least understand what the performance impact of this change is.

Can you do this profiling and post your findings on this PR?

Reply:
That's interesting... Do you have ready to use profilers ?

Reply:
Not that I am aware of. I think to try this out, it would probably be best to use `npm link` in a project that is setting the `NODE_ENV` environment variable to production and minifying it out properly (e.g. in conjunction with [webpack's DefinePlugin](https://webpack.js.org/plugins/define-plugin/)).